app/Http/Controllers/Controller.php

```diff
@@ -99,7 +99,14 @@ public function home()
             'stream' => 'Stream',
         ];
         $options = [
-            'legend' => 'Meeting Types Legend',
+            'legend' => [
+                'label' => 'Meeting Types Legend',
+                'checked' => false,
+                ],
```

Comment:
would be great to get format with PSR-12

Reply:
One of these days, I'm gonna get all my environments right! Thanks for catching!